data/tensorflow/operator/recordio_dataset_op.cc

```diff
@@ -0,0 +1,215 @@
+#include "tensorflow/core/framework/dataset.h"
+#include "tensorflow/core/framework/common_shape_fns.h"
+#include "../recordio/recordio_reader.h"
```

Comment:
don't use relative path like .. if needed could add path to HEADER_DIR

Reply:
fixed.